partner_identification_dea/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright (C) 2021 Open Source Integrators
```

Comment:
We should not have copyright lines in init files (and make all license lines https:).